docs/generated/manifests/cloud.json

```diff
@@ -40,16 +30,6 @@
     "path": "/nx-cloud/intro/ci-with-nx",
     "tags": []
   },
-  "/nx-cloud/intro/nx-cloud-workflows": {
-    "id": "nx-cloud-workflows",
-    "name": "Nx Cloud Workflows",
-    "description": "",
-    "file": "nx-cloud/intro/nx-cloud-workflows",
-    "itemList": [],
-    "isExternal": false,
-    "path": "/nx-cloud/intro/nx-cloud-workflows",
-    "tags": []
-  },
   "/nx-cloud/features": {
```

Comment:
I'd call this section "Core Features" just like we do on the Nx section.

Reply:
In the Nx section, we call it Core Features to distinguish them from Plugin Features.  With the line being blurred between plugin features and core features with project config v3, I think we should start to move away from the Core/Plugin distinction and eventually flatten the Nx feature list to all be under a single Features list.

That's why I think the Nx Cloud/CI features should be called Features and not Core Features